content/SCALE/GettingStarted/Migrate/MigratingFromCORE.md

```diff
@@ -93,19 +93,19 @@ If this process fails, retry using the iso file method above.
 
    Then click **APPLY UPDATE**.
 
-7. After the update completes, reboot the system if it does not reboot automatically.
+7. After the update completes, restart the system if it does not restart automatically.
 
-   {{< trueimage src="/images/SCALE/SystemSettings/SidegradeRestart.png" alt="Reboot to Finish" id="Reboot to Finish" >}}
+   {{< trueimage src="/images/SCALE/SystemSettings/SidegradeRestart.png" alt="restart to Finish" id="restart to Finish" >}}
 
-After TrueNAS reboots, you might need to [use the Console Setup menu to configure the primary networking interfaces]({{< relref "ConsoleSetupMenuScale.md" >}}) to enable GUI accessibility.
+After TrueNAS restarts, you might need to [use the Console Setup menu to configure the primary networking interfaces]({{< relref "ConsoleSetupMenuScale.md" >}}) to enable GUI accessibility.
```

Comment:
I do not know why we did this in the original sentence, so here's a chance to fix this... 
```suggestion
After TrueNAS restarts, to enable GUI accessibility, you might need to use the [Console Setup menu]({{< relref "ConsoleSetupMenuScale.md" >}}) to configure the primary networking interfaces.
```